backend/api/admin/src/main/java/co/airy/core/api/admin/payload/WebhookListResponsePayload.java

```diff
@@ -10,5 +10,5 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class WebhookListResponsePayload {
-    private List<WebhookResponsePayload> data;
+    private List<WebhookListPayload> data;
```

Comment:
Why can we not use `WebhookResponsePayload` here?

Reply:
That's true and I think in this PR we can leave it like that. 

But for a more general discussion: Currently when you don't set a status on `WebhookResponsePayload` the status is returned as:
```json
{
"status": null
}
```

@AitorAlgorta As far as I can recall we had introduced this default in the old system to make it easier for mobile clients to parse payloads. Perhaps it's worth revisiting this discussion because in my opinion, (and it's just taste I guess) `null` values on JSON payloads add more clutter than information. It would also allow us to recycle payloads as in the example above.

cc @abhi-airy @AitorAlgorta @juan-sebastian what's your opinion on this default? Should we consider changing it? If so we can open a new ticket for discussion and implementation :)

Reply:
For me the response should not include `null` values (see [ignore null values](https://www.baeldung.com/jackson-ignore-null-fields)), the fort-end can assume that a missing value is a `null` value. This way the response body is not **clutter** as you said.